mne/decoding/transformer.py

```diff
@@ -107,7 +107,7 @@ def transform(self, epochs_data, y=None):
             raise ValueError("epochs_data should be of type ndarray (got %s)."
                              % type(epochs_data))
 
-        X = np.atleast_3d(epochs_data)
+        X = np.atleast_3d(epochs_data).copy()
```

Comment:
you should add a `Notes` section or say in the `epochs_data` description that it will be copied, because this could be a big memory operation